lldb/test/API/tools/lldb-dap/io/TestDAP_io.py

```diff
@@ -48,18 +48,18 @@ def test_invalid_header(self):
         lldb-dap handles invalid message headers.
         """
         process = self.launch()
-        process.stdin.write(b"not the corret message header")
+        process.stdin.write(b"not the correct message header")
         process.stdin.close()
-        self.assertEqual(process.wait(timeout=5.0), 1)
```

Comment:
could you quickly explain what these 1 and 0 means in both versions of the test?